playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -116,3 +116,12 @@
   template:
     src: clouds.yml.j2
     dest: /etc/openstack/clouds.yaml
+
+- name: Save Shipyard env variables to ~/.airshiprc
+  blockinfile:
+    path: "{{ ansible_user_dir }}/.airshiprc"
```

Comment:
I would rather use the workspace instead. ``socok8s_workspace`` so that we continue to be self contained.

Reply:
workspace is on localhost, but I want this on deployer

Reply:
Got it.